x-pack/plugins/security_solution/public/resolver/models/process_event.test.ts

```diff
@@ -56,14 +60,14 @@ describe('process event', () => {
       // each event has a unique id, a through h
       // order is arbitrary
       events = [
-        mock(-1, 'a'),
-        mock(0, 'c'),
-        mock(1, 'e'),
-        mock(NaN, 'g'),
-        mock(-1, 'b'),
-        mock(0, 'd'),
-        mock(1, 'f'),
-        mock(NaN, 'h'),
+        mock(-1, 'a') as SafeResolverEvent,
```

Comment:
We should create a ticket to handle this. The process of switching to safe types needs to be gradual, but let's not forget to finish.